app/views/items/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="container">
+  <div class="row justify-content-center">
+    <div class="col-6 shadow">
+      <%= simple_form_for @item do |f| %>
+        <%= f.input :name %>
+        <%= f.select :type, collection: Item::TYPES %>
```

Comment:
item_type

Reply:
Updated! Refer to changes in latest commits. Thanks!